### PR TITLE
new package: mingw-w64-xournalpp

### PR DESCRIPTION
**Pull request by Kreijstal**
new Xournal++ package

### CODE REVIEW

**Comment by Biswa96** [comment]
Thank you for the contribution. May I suggest to follow the syntax from mingw-w64-PKGBUILD-templates/PKGBUILD.CMake-tarball.sh file? For example, use ninja and cmake commands, use source tarball etc.

File: mingw-w64-xournal++/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+         "${MINGW_PACKAGE_PREFIX}-libzip"
+         "${MINGW_PACKAGE_PREFIX}-lua"
+         "${MINGW_PACKAGE_PREFIX}-portaudio")
+makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
+             "${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-libbacktrace"
+             "${MINGW_PACKAGE_PREFIX}-ninja"
+             "${MINGW_PACKAGE_PREFIX}-gtksourceview4"
+             "${MINGW_PACKAGE_PREFIX}-imagemagick"
+             'patch'
```
Comment (Biswa96) [inline mingw-w64-xournal++/PKGBUILD:25]: patch, make, git are not required.

File: mingw-w64-xournal++/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Maintainer: kreijstal <kreijstal@hotmail.com>
+
+_realname=xournalpp
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.2.2
+pkgrel=1
+pkgdesc="Xournal++ - a modern handwritten note-taking software"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64' 'mingw32' 'clang32')
```
Comment (Biswa96) [inline mingw-w64-xournal++/PKGBUILD:10]: Please remove 32 bit arch. https://www.msys2.org/news/#2023-12-13-starting-to-drop-some-32-bit-packages

**Comment by Kreijstal** [comment]
did I overlook something?

**Comment by ognevny** [comment]
> did I overlook something?

use `build-${MSYSTEM}` directory for build. for details see the [template](https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-PKGBUILD-templates%2FPKGBUILD.CMake-tarball.sh#L32)

**Comment by Kreijstal** [comment]
> > did I overlook something?
> 
> use `build-${MSYSTEM}` directory for build. for details see the [template](https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-PKGBUILD-templates%2FPKGBUILD.CMake-tarball.sh#L32)

Done :)

**Comment by ognevny** [comment]
I forgot to check it. lgtm

**Comment by Biswa96** [comment]
I just noticed that many things were wrong with this package - PKGBUILD directory, dependencies, license, metadata etc. I shall create a separate PR to fix those.

**Comment by ognevny** [comment]
> I just noticed that many things were wrong with this package - PKGBUILD directory, dependencies, license, metadata etc. I shall create a separate PR to fix those.

oh, oops. sorry

**Pull request merged**